### PR TITLE
Treeview and List: Add localization for item checkBoxes aria-label value 'Check State' (T1247518)

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/js/localization/messages/en.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
         "dxList-listAriaLabel-deletable": "Deletable items",
         "dxListEditDecorator-delete": "Delete",
         "dxListEditDecorator-more": "More",
+        "dxList-checkState": "Check State",
```
Comment (vladaskorohodova) [inline packages/devextreme/js/localization/messages/en.json]:
```suggestion
        "dxList-checkState": "Check state",
```

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/en.json]: [24621d1](https://github.com/DevExpress/DevExtreme/pull/28071/commits/24621d14ab11918f6ace4f4a9ebf40cdb4ffd9b5)

**Review by ksercs — DISMISSED** [review]

**Review by vladaskorohodova — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.checkboxes.tests.js [hunk 2]
```diff
@@ -318,6 +319,20 @@ QUnit.test('checkbox should have aria-label="Check State" attribute', function(a
     assert.strictEqual($checkbox.attr('aria-label'), 'Check State');
 });
 
+QUnit.test('checkbox should have a correct aria-label value based on localiztion (T1247518)', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.checkboxes.tests.js]:
```suggestion
QUnit.test('checkbox should have a correct aria-label value based on localization (T1247518)', function(assert) {
```

**Review by ksercs — APPROVED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.checkboxes.tests.js (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.checkboxes.tests.js]: [0dada4e](https://github.com/DevExpress/DevExtreme/pull/28071/commits/0dada4e69c01ce0b322064f04770fb004fa26aab)

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**